intern/resource/boss/resource.py

```diff
@@ -442,6 +459,15 @@ def __init__(self, name, collection_name, experiment_name, type="image",
         self.exp_name = experiment_name
 
         self._type = self.validate_type(type)
+
+        # Class is considered fully initialized if datatype set during
+        # initialization.
+        if datatype == '':
```

Comment:
I was looking at this as meaning that the bare minimum is specified so the ChannelResource could be used for a cutout. Maybe the property should be renamed to `channel_cutout_ready` so the meaning is clearer?